content/html/concepts/elements/terms/s/s.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '<s>'
+Description: 'Used to draw attention to one or more words, usually rendered in strike-through text.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Elements'
+  - 'Tags'
+  - 'Style'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<s>`** element is used to highlight no longer pertinent, inaccurate, or wrong text. Browsers render it showing strike-through text.
+
+## Syntax
+
+```pseudo
+<s>
+  <!-- No longer pertinent, inaccurate, or wrong text here -->
+</s>
+```
+
+The `<s>` element should only contain text content or other elements meant for text content.
+
+> **Note:** The `<s>` element is semantically different from the `<del>` element because the former is just an edit while the latter conveys an actual update in a document.
```

Comment:
```suggestion
> **Note:** The `<s>` element is semantically different from the [`<del>`](https://www.codecademy.com/resources/docs/html/elements/del) element because the former is just an edit while the latter conveys an actual update in a document.
```

Reply:
Done